java/vector/src/test/java/org/apache/arrow/vector/ipc/TestArrowStream.java

```diff
@@ -44,8 +44,9 @@ public void testEmptyStream() throws IOException {
 
     // Write the stream.
     ByteArrayOutputStream out = new ByteArrayOutputStream();
-    try (ArrowStreamWriter writer = new ArrowStreamWriter(root, null, out)) {
-    }
+    ArrowStreamWriter writer = new ArrowStreamWriter(root, null, out);
+    writer.close();
+    Assert.assertTrue(out.size() > 0);
```

Comment:
Just curious, what prevents:

```
try (ArrowStreamWriter writer = new ArrowStreamWriter(root, null, out)) {
}
```
?

Not a big deal but I think the code before prevents `writer` from leaking to outer scope.

Reply:
It's from `EmptyBlock` with `LITERAL_TRY`.

Reply:
http://checkstyle.sourceforge.net/config_blocks.html#EmptyBlock